source/server/lds_api.cc

```diff
@@ -43,11 +43,11 @@ void LdsApiImpl::onConfigUpdate(
     const std::string& system_version_info) {
   std::unique_ptr<Cleanup> maybe_eds_resume;
   if (cm_.adsMux()) {
-    const auto type_url = Config::getTypeUrl<envoy::config::route::v3::RouteConfiguration>(
-        envoy::config::core::v3::ApiVersion::V2);
-    cm_.adsMux()->pause(type_url);
+    const auto type_urls =
+        Config::getAllVersionTypeUrls<envoy::config::route::v3::RouteConfiguration>();
+    cm_.adsMux()->pause(type_urls);
     maybe_eds_resume =
```

Comment:
nit: not changes in this  PR but /s/maybe_eds_resume/maybe_rds_resume ?

Reply:
good catch, thanks!